packages/mdc-fab/mdc-fab.scss

```diff
@@ -12,98 +12,58 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-@import "@material/animation/functions";
-@import "@material/animation/variables";
-@import "@material/elevation/mixins";
-@import "@material/ripple/mixins";
-@import "@material/theme/mixins";
+@import "@material/theme/variables";
+@import "./mixins";
 
 // postcss-bem-linter: define fab
-.mdc-fab {
-  @include mdc-ripple-base;
-  @include mdc-ripple-bg((pseudo: "::before", base-color: white, opacity: .16));
-  @include mdc-ripple-fg((pseudo: "::after", base-color: white, opacity: .16));
-
-  display: inline-flex;
-  position: relative;
-  justify-content: center;
-  width: 56px;
-  height: 56px;
-  padding: 0;
-  transition: box-shadow 280ms $mdc-animation-standard-curve-timing-function;
-  border: none;
-  border-radius: 50%;
-  cursor: pointer;
-  user-select: none;
-  box-sizing: border-box;
-  fill: currentColor;
-  -moz-appearance: none;
-  -webkit-appearance: none;
-  overflow: hidden;
-
-  &:not(.mdc-ripple-upgraded) {
-    -webkit-tap-highlight-color: rgba(0, 0, 0, .18);
-  }
-
-  @include mdc-theme-prop(background-color, secondary);
-  @include mdc-theme-prop(color, text-primary-on-secondary);
-  @include mdc-elevation(6);
 
-  &--mini {
-    width: 40px;
-    height: 40px;
-  }
-
-  &--plain {
-    background-color: white;
-
-    @include mdc-theme-prop(color, text-primary-on-light);
-    @include mdc-ripple-bg((pseudo: "::before"));
-    @include mdc-ripple-fg((pseudo: "::after"));
-  }
-
-  &:active,
-  &:focus {
-    outline: none;
-  }
+$mdc-theme-property-values-custom: map-merge(
```

Comment:
Removed property map after discussing with the team offline. We will now allow directly passing arbitrary color values instead.

Tap highlight color should probably be computed, but I'd prefer to do that in a separate PR to keep this one small and focused.